app/controllers/citizens/dependants/new_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Citizens
+  module Dependants
+    class NewController < BaseController
```

Comment:
The pattern I used in vehicles was to have a `VehiclesController` that created the main object, and then the controllers that modified vehicles would be in the `Vehicles` namespace. So `Vehicles::FooController`.

`NewController` feels a little strange. Would making this `Citizens::DependantsController` be simpler?

Also - the `dependents` step prefix wouldn't be necessary in this controller. It would just have the step name `:dependents`, and `Dependents::DetailsController` would stay as `dependents_details` (it would need the prefix).